src/components/AlertBanner/__stories__/alertBanner.stories.scss

```diff
@@ -18,14 +18,6 @@
   }
 
   &_pink {
-    .monday-alert-banner__close-button-wrapper {
-      background: var(--private-color);
-    }
-    .monday-alert-banner__inner {
-      background: var(--private-color);
-    }
-    .monday-alert-banner__alert-banner-close-btn {
-      background: var(--private-color);
-    }
+    background: var(--sb-color-sofia_pink);
```

Comment:
there were three different classes before, is this covers them all?
just making sure this is intentional and not a testing leftover or a tech debt 😄